config.w32

```diff
@@ -59,29 +57,18 @@ if(PHP_SOLR != 'no')
 			PHP_SOLR = "no";
 		}
 	}
-	if(PHP_JSON == 'no' || !ADD_EXTENSION_DEP('solr', 'json')) {
-		WARNING('solr was not enabled; json is not enabled');
-		PHP_SOLR = "no";
-	}
```

Comment:
I'm not sure about this. JSON is not guaranteed to be available prior to PHP 8.0.0.

Reply:
I remember having to remove these lines, otherwise it would not compile. That was probably before PHP 8.0.0RC3
https://github.com/php/php-src/commit/560970138915791fe29380c8f31a7ec18379b00f

Reply:
New PR, replacing this one: https://github.com/php/pecl-search_engine-solr/pull/34